app/models/user.rb

```diff
@@ -400,6 +400,16 @@ def va_profile_email
     vet360_contact_info&.email&.email_address
   end
 
+  # Created for spec testing V2
```

Comment:
@LindseySaari @RachalCassity I'd prefer if you all wait for Identity team approval when files that we own are changed.

Also, I might be misinterpreting based off the comments for these functions, but we really shouldn't be committing functions for spec testing in the production code.

Reply:
Sorry, these are for the new service as well. User's vet360_contact_info is dependent on VAProfileRedis::ContactInformation. To avoid any issues with mobile and prefill, vaprofile_contact_info and va_profile_v2_email replaces the exisiting vet360_contact_info.